llarp/iwp/linklayer.cpp

```diff
@@ -90,6 +90,7 @@ namespace llarp
       }
       else
       {
+        Lock lock(&m_AuthedLinksMutex);
```

Comment:
I thought we said this lock was not needed on discord?

Reply:
We use the null lock/null mutex here now.